src/index.html

```diff
@@ -1,10 +1,22 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <script>
-        var subdomain = window.location.host.split('.')[0];
-         window.location = 'https://' + subdomain + '.globalforestwatch.org/home/';
-    </script>
+  <!-- Hotjar Tracking Code for https://fires.globalforestwatch.org/ -->
+  <script>
+    (function(h,o,t,j,a,r) {
+      h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+      h._hjSettings={hjid:1494735,hjsv:6};
+      a=o.getElementsByTagName('head')[0];
+      r=o.createElement('script');r.async=1;
+      r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+      a.appendChild(r);
+      })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
+  </script>
+
```

Comment:
PDW eric, there's no way to test this on our end. Once we push to staging, we'll have the client test it and report back to us.